src/DebugEngineHost/HostConfigurationStore.cs

```diff
@@ -110,26 +109,22 @@ public HostLogger GetLogger(string enableLoggingSettingName, string logFileName)
                 !(enableLoggingValue is int) ||
                 ((int)enableLoggingValue) == 0)
             {
-                return null;
+                return new HostLogger();    // empty logger
```

Comment:
No HostConfigurationStore is available on which to call GetLoggerFromCmd. I will move this functionality directly into the HostLogger.